scripts/PMID_downloader.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from Bio import Entrez
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import time
+
+#This is for using the PubMed API
+Entrez.email = "patrick-simon.nairne@bih-charite.de"
+
+#This function takes the path of a .txt file with lines of the form PMID_1234567
+#and returns a list of the form [1234567,...]
+def pmid_list_maker(txt_file_path):
+    pmid_list = []
```

Comment:
In python you would write a doc string like:

```suggestion
def pmid_list_maker(txt_file_path):
"""This function takes the path of a .txt file with lines of the form PMID_1234567 and returns a list of the form [1234567,...]"""
    pmid_list = []
```
```suggestion
#This function takes the path of a .txt file with lines of the form PMID_1234567
#and returns a list of the form [1234567,...]
def pmid_list_maker(txt_file_path):
    pmid_list = []
```